extensions/filebeat/7.x/wazuh-module/README.md

```diff
@@ -56,6 +56,7 @@ To add a new version of the module it is necessary to follow the following steps
 
 ```
 # chmod 755 wazuh
+# chmod 755 wazuh/_meta
```

Comment:
This file shouldn't be modified here. It is not within the manager's scope.

Reply:
Additionally, there are other files related to the Wazuh agent that should be removed from this pull request. PTAL.